Tests/SSL/test_ssl_containers.py

```diff
@@ -634,7 +634,24 @@ def test_simclr_dataloader_type() -> None:
     """ This test checks if the transform pipeline of a SSL job can handle different
     data types coming from the dataloader.
     """
-    def check_types_in_dataloader(dataloader: CombinedLoader) -> None:
+    # TODO: Once the pytorch lightning bug is fixed the following test can be removed.
+    # The training and val loader will be both CombinedLoaders
+    def check_types_in_train_dataloader(dataloader: dict) -> None:
+        for i, batch in enumerate(dataloader[SSLDataModuleType.ENCODER]):
+            assert isinstance(batch[0][0], torch.Tensor)
+            assert isinstance(batch[0][1], torch.Tensor)
+            assert isinstance(batch[1], torch.Tensor)
+            if i == 1:
+                break
+
+        for i, batch in enumerate(dataloader[SSLDataModuleType.LINEAR_HEAD]):
+            assert isinstance(batch[0], torch.Tensor)
+            assert isinstance(batch[1], torch.Tensor)
+            assert isinstance(batch[2], torch.Tensor)
+            if i == 1:
+                break
```

Comment:
I guess this is an addition from Anton or Max to avoid a separate PR :) ?

Reply:
yes, correct